backend/payments/tests/test_payments_serializer.py

```
@@ -0,0 +1,132 @@
+    assert "receipt_number" in s.errors
+
+
+@pytest.mark.django_db
+def test_update_posted_raises_validation_error():
+    inst = PaymentFactory(status="posted")
+    factory = APIRequestFactory()
+    req = factory.patch(f"/api/v1/payments/{inst.id}/")
+    req.user = UserFactory()
+    s = PaymentSerializer(instance=inst, data={"amount": "200.00"}, partial=True, context={"request": req})
+    assert s.is_valid(), s.errors
+    with pytest.raises(Exception):
```

Ruff (0.14.4)

Do not assert blind exception: `Exception`

(B017)